hilti/runtime/include/fiber.h

```diff
@@ -1,8 +1,5 @@
 // Copyright (c) 2020 by the Zeek Project. See LICENSE for details.
 //
-// TODO(robin): This may not be the most efficient solution for implementing fibers
-// yet (due to using std::function, std::any)
-//
 // There are also at least two proposals for upcoming version of C++ that could implement this:
 //      - Coroutunes: http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2017/n4680.pdf
```

Comment:
We can remove this, too (co-routines won't work for us, and fibers aren't on the horizon).